test/extensions/http/header_validators/envoy_default/path_normalizer_fuzz.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package test.extensions.http.header_validators.envoy_default;
+
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+
+import "envoy/extensions/http/header_validators/envoy_default/v3/header_validator.proto";
+
+message PathNormalizerFuzzTestCase {
+  // the character set of the :path and :method headers is validated before normalization.
+  // As such it is ok to use strings to represent these values in the fuzz test.
+  string method = 1;
+  string path = 2;
+
+  envoy.extensions.http.header_validators.envoy_default.v3.HeaderValidatorConfig config = 3;
```

Comment:
What's the reason we opted for the encapping message vs UriPathNormalizationOptions? The latter seems to better target the space of this fuzzer.

Reply:
You are correct. Fixed.